### DISCUSSION TITLE
Cretor Coin algorithm brainstorming?

### DISCUSSION THREAD

**Discussion by ghost** [deso-protocol/dips#70] [Medium CIPs]
In addition to the existing Cretor Coin algorithm, an Initial Coin Offering-like feature can be added.

This algorithm: it will be just like ICO. Creator will be able to set certain limits before selling his coins. For example, 1 Creator Coin will be sold as 0.01 Clout. The total amount of Creator Coins will be 10K. After all Creator Coins are sold, the price will be determined by swap.

In this way, Bitclout will also be on a platform like a decentralized exchange. The value of Clout can be further increased by adding features such as staking and farming.

Another perspective, Any company will be able to collect investments without the need for technical knowledge. Currently, ICO costs at least 10K dollars. Since Bitclout is a social media platform, companies will be able to make their promotions here.

**Comment by arhebbar** [comment]
Love this. This could also be a way to take care of the bonding curve related pump and dumps and scams as well. Definitely an idea a node built for Creators or even an ICO specific node could pick up if the core node doesn't pick up.
